config/jobs/kubernetes/sig-windows/windows-gce.yaml

```diff
@@ -227,6 +245,51 @@ periodics:
     testgrid-tab-name: gce-windows-containerd-master
     description: Runs tests on a Kubernetes cluster with Windows containerd nodes on GCE
 
+- name: ci-kubernetes-e2e-windows-node-throughput
+  interval: 1h
```

Comment:
About how long does it take for a load test run to complete?

I see that --timeout=40m is set below, which leaves another 20m for cluster setup and teardown before test runs will start to overlap/timeout. Consider if we should bump this to 90m or 2h.

Reply:
For 80 pods, it takes around 10mins. 1h should be enough, but will bump to 2h just in case.